httpie/output/formatters/xml.py

```diff
@@ -26,9 +40,18 @@ def pretty_xml(document: 'Document',
     if standalone is not None and sys.version_info >= (3, 9):
```

Comment:
If we indeed need this, then it would be nice to move the version check to `compat.py`.